tango_ros_common/tango_ros_native/CMakeLists.txt

```diff
@@ -14,7 +14,10 @@ find_package(catkin REQUIRED COMPONENTS
    )
 
 find_package(cmake_modules REQUIRED)
-find_package(OpenCV 3 REQUIRED)
+find_package(OpenCV 2 QUIET)
```

Comment:
I will try it out and submit the changes, I think it looks a bit better.
Thanks @smits !